airbyte-integrations/connectors/destination-bigquery/src/test-integration/kotlin/io/airbyte/integrations/destination/bigquery/BigqueryWriteTest.kt

```diff
@@ -87,7 +86,7 @@ abstract class BigqueryTDWriteTest(configContents: String) :
             nestedFloatLosesPrecision = true,
             integerCanBeLarge = false,
             numberCanBeLarge = false,
-            timeWithTimezoneBehavior = SimpleValueBehavior.PASS_THROUGH,
+            timeWithTimezoneBehavior = SimpleValueBehavior.STRONGLY_TYPE,
```

Comment:
previously we relied on bigquery to do validation... but we we're writing timetz values as STRING, since bigquery doesn't have a timetz column. So they were never nulled out.

Now we can validate values in-connector, so we can correctly drop invalid timetz values.